spath.sh

```diff
@@ -33,7 +33,12 @@ fi
 
 if ! echo $LD_LIBRARY_PATH | grep -q deps/lib
 then
-    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$prefix/deps/lib"
+    if test -n "$LD_LIBRARY_PATH"
+    then
+        export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$prefix/deps/lib"
+    else
+        export LD_LIBRARY_PATH="$prefix/deps/lib"
+    fi
     test $verbose && echo "LD_LIBRARY_PATH: $LD_LIBRARY_PATH"
 else
     test $verbose && echo "LD_LIBRARY_PATH: already set"
```

Comment:
I think we should drop the "already set" bs and just do this

```bash 
test` $verbose && echo "LD_LIBRARY_PATH: $LD_LIBRARY_PATH"
```
Once, after the adjustment has (or has not) been made.